docs/OV_Runtime_UG/deployment/local-distribution.md

```diff
@@ -20,7 +20,7 @@ The `plugins.xml` file with information about inference devices must also be tak
 
 The picture below presents dependencies between the OpenVINO Runtime core and pluggable libraries:
 
-![deployment_full]
+![](../../img/deployment_full.svg)
```

Comment:
can we use more traditional color for the picture?

Reply:
Hi :) I used colors from brand color guideline to fit with colors of the original picture. If You suggest to change them, then which colors will fit the most?

Reply:
Color like here https://docs.openvino.ai/latest/ovms_what_is_openvino_model_server.html

Reply:
I committed the new color version. I have to use colors from brand color guideline, so these are the closest to the ones from what You linked.